nx-os/poap/poap.py

```diff
@@ -275,21 +275,6 @@ def abort_cleanup_exit () :
 import signal
 import string
```

Comment:
Might be a good idea to take this refactor as an opportunity to move these imports to the top of the source